From 101ccd0754da7a873c03ed84c5c67a638bb1c70b Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 27 Sep 2021 09:33:48 +0000
Subject: IWYU: add memory for std::unique_ptr in ResourceBundleSourceMap

Bug: None
Change-Id: I6ba4ecfbe2883c572bf5f5fbce819d6e3c749c22
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3184742
Commit-Queue: Jochen Eisinger <jochen@chromium.org>
Reviewed-by: Jochen Eisinger <jochen@chromium.org>
Cr-Commit-Position: refs/heads/main@{#925199}
---
 extensions/renderer/resource_bundle_source_map.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/extensions/renderer/resource_bundle_source_map.h b/extensions/renderer/resource_bundle_source_map.h
index 4e368c5a9ffd7..dd98ad78f6960 100644
--- a/extensions/renderer/resource_bundle_source_map.h
+++ b/extensions/renderer/resource_bundle_source_map.h
@@ -6,6 +6,7 @@
 #define EXTENSIONS_RENDERER_RESOURCE_BUNDLE_SOURCE_MAP_H_
 
 #include <map>
+#include <memory>
 #include <string>
 
 #include "base/macros.h"
-- 
2.30.2

